### PR TITLE
Write should always end with a newline

### PR DESCRIPTION
**Pull request by ruudk**
When modifying an ini file I noticed the file did not end with a newline. Not sure if this was done intentionally, but this changes that.

### CODE REVIEW

File: godotenv.go [hunk 1]
```diff
@@ -148,7 +148,7 @@ func Exec(filenames []string, cmd string, cmdArgs []string) error {
 
 // Write serializes the given environment and writes it to a file
 func Write(envMap map[string]string, filename string) error {
-	content, err := Marshal(envMap)
+	content, err := Marshal(envMap) + "\n"
```
Comment (gnarlex) [inline godotenv.go:151]: This doesn't compile as `Marshal(envMap)` returns two values.

Comment (ruudk) [reply godotenv.go]: Oef, you're right. I did this one too quickly via GitHub website. Changed it now. Better?

**Review by gnarlex — APPROVED** [review]

**Comment by ruudk** [comment]
I think this is still relevant. Just noticed that a script was broken. I have a go program that modifies .env files, and then another script tried to append to the modified .env file, assuming that every file is always ended with a newline. Since the go program didn't write the newline, the appended env var was added to the value of another key.

**Comment by ruudk** [comment]
I see this is already handled in https://github.com/joho/godotenv/pull/133

**Pull request closed without merging**